docs/learning/tutorial.soy

```diff
@@ -126,6 +162,52 @@ echo "android_library(
 </blockquote>
 </p>
 
+<span><block class="mac linux android kotlin" /></span>
+<p>
+  Now that Buck can locate your Android SDK, it is time to compile some Java code. First, we create a simple <code>Activity</code> at <code>kotlin/com/example/activity/MyFirstActivity.kt</code>:
+</p>
+
+{literal}<pre>
+mkdir -p kotlin/com/example/activity/
+echo "package com.example.activity
+
+import android.app.Activity
+import android.os.Bundle
+
+class MyFirstActivity : Activity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+}" > kotlin/com/example/activity/MyFirstActivity.kt
+</pre>{/literal}
+
+<p>
+  Now we need a build file that defines a build rule to compile this Java code, so we create an <code><a href="rule/android_library.html">android_library()</a></code> rule in <code>java/com/example/activity/BUCK</code>:
```

Comment:
`java/com/example/activity/BUCK` should be `kotlin` right?

Reply:
Fixed.